docs-src/content/using-cubic-sdk/client-configs.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "Recognition Configurations"
+weight: 25
+---
+
+An in-depth explanation of the various fields of the complete SDK can be found [here](../../protobuf/autogen-doc-cubic-proto/).  The sub-section [RecognitionConfig](../../protobuf/autogen-doc-cubic-proto/#message-recognitionconfig) is particularly important here.
+
+This page here discusses the more common combinations sent to the server.
+
+## Fields
+
+Here's a quick refresher of the fields.
```

Comment:
"refresher" implies the reader should already know about the fields. Is this the case? (I don't know if there's another page describing the configs that the reader will see before this.) If not, change it to "quick overview".